pyresample/geometry.py

```diff
@@ -2587,11 +2587,9 @@ def get_area_slices(self, area_to_cover, shape_divisible_by=None):
             return x_slice, y_slice
 
         if not self.is_geostationary:
-            raise NotImplementedError("Source projection must be 'geos' if "
-                                      "source/target projections are not "
-                                      "equal.")
-
-        data_boundary = Boundary(*get_geostationary_bounding_box_in_lonlats(self))
+            data_boundary = self.boundary(frequency=100)
```

Comment:
what is the rationale for choosing `100` here? (we also hardcode it a bit later https://github.com/pytroll/pyresample/blob/4e245a906bf3aeff072f6b61febdd1df298fb425/pyresample/geometry.py#L2597). I see that the geos case has some more handling of the `frequency` here https://github.com/pytroll/pyresample/blob/4e245a906bf3aeff072f6b61febdd1df298fb425/pyresample/geometry.py#L1559-L1569.